src/svgs/icon-clear-small.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Icons/Zz Working/X/Clear small</title>
+    <g id="Icons/Zz-Working/X/Clear-small" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" fill-opacity="0.38">
+        <path d="M5.61289944,4.20970461 L5.70710678,4.29289322 L8,6.585 L10.2928932,4.29289322 C10.6834175,3.90236893 11.3165825,3.90236893 11.7071068,4.29289322 C12.0675907,4.65337718 12.0953203,5.22060824 11.7902954,5.61289944 L11.7071068,5.70710678 L9.415,8 L11.7071068,10.2928932 C12.0976311,10.6834175 12.0976311,11.3165825 11.7071068,11.7071068 C11.3466228,12.0675907 10.7793918,12.0953203 10.3871006,11.7902954 L10.2928932,11.7071068 L8,9.415 L5.70710678,11.7071068 C5.31658249,12.0976311 4.68341751,12.0976311 4.29289322,11.7071068 C3.93240926,11.3466228 3.90467972,10.7793918 4.20970461,10.3871006 L4.29289322,10.2928932 L6.585,8 L4.29289322,5.70710678 C3.90236893,5.31658249 3.90236893,4.68341751 4.29289322,4.29289322 C4.62333685,3.96244959 5.12750547,3.91161211 5.51140295,4.14038077 L5.61289944,4.20970461 Z" id="Path" fill="#000000" fill-rule="nonzero"></path>
+    </g>
+</svg>
```

Comment:
Do we need newlines at the end of svg files? 🤔

Reply:
I gave the good fight. I think we want a linter if we want it to be required.